proto/regen/group/v1alpha1/tx.proto

```diff
@@ -1,300 +0,0 @@
-syntax = "proto3";
-
-package regen.group.v1alpha1;
-
-option go_package = "github.com/regen-network/regen-ledger/x/group";
```

Comment:
On second thought, we should probably keep the proto files for reference as long as the experimental build includes the group module, allowing users of the experimental test network to look up functionality.

We can remove this line from each group proto file to prevent codegen.